e2e/sample.config.yaml

```diff
@@ -30,7 +30,7 @@ relayers:
     tag: "latest"
   - id: hyperspace
     image: ghcr.io/misko9/hyperspace
-    tag: "local"
+    tag: "timeout"
```

Comment:
It is a bit weird that the tag is called timeout :P

Reply:
yeah it is a bit weird, maybe for now we can re-tag this image and give it a better name. Happy if this happened in a follow up though.